app/src/main/res/layout/fragment_media_detail.xml

```diff
@@ -52,6 +52,34 @@
                 android:orientation="vertical"
                 android:padding="@dimen/standard_gap">
 
+                <LinearLayout
```

Comment:
Please verify the formatting

Reply:
@ashishkumar468 I didn't get the comment

Reply:
I meant the code-style, the formatting